examples/mnist/thirdparty/CMakeLists.txt

```diff
@@ -0,0 +1,59 @@
+#
```

Comment:
/usr/local/share/modelbox/tools/create -flowunit -name "flowunit" -lang python

/usr/local/share/modelbox/tools/create -project -name="project" -rootpath [] -path []


{
    "project": null,
    "rootpath": "path",
    "path": "path",
    "name": "name",
}



/usr/local/share/modelbox/tools/create -flowunit -name "flowunit" -lang python --input name=a,device=cpu,type=float32 --input name=a,device=cpu,type=float32
{
    "flowunit": null,
    "name": "path",
    "lang": "path",
    "name": "name",
    "input": [
        {
        "name": name,
        "device": device,
        "type": float32,
        },
        {
        "name": name,
        "device": device,
        "type": float32,
        },
    ]
}


for (k, v : json:values) {
    if (v == null) {
        cmd.pushback(k)
        continue;
    }

    if (!v.array && !v.value) {
        error;
        break;
    }


    if (v.value) {
        cmd.pushback("-" +k + "=" + v)
        continue;
    }
    
    if (v.array) {
        for () {
            v="name=xx,bb,cc"
        }
    }

    cmd.pushback("-" +k + "=" + v)
}